scripts/setup.sh

```diff
@@ -100,3 +100,47 @@ else
   # Mac OS.
   export CHROME_BIN="/Applications/Google Chrome.app/Contents/MacOS/Google Chrome"
 fi
+
+# Check the version of Python. If its version does not start with 2.7, then
+# try and see whether the 'python2.7' command exists.
+# Borrowed from bhenning's solution for checking python version
+# http://stackoverflow.com/questions/592620
+test_python_version(){
+  EXPECTED_PYTHON_VERSION_PREFIX="2.7"
+  PYTHON_VERSION=$($1 --version 2>&1)
+  if [[ $PYTHON_VERSION =~ Python[[:space:]](.+) ]]; then
+    PYTHON_VERSION=${BASH_REMATCH[1]}
+  else
+    echo "Unrecognizable Python command output: ${PYTHON_VERSION}"
+  fi
+  if [[ "${PYTHON_VERSION}" = "${EXPECTED_PYTHON_VERSION_PREFIX}*" ]]; then
+    #false
```

Comment:
Commands should have a space after the '#'. Also, they should form complete thoughts and be a sentence, such as:

```# The value '1' indicates a true return value, indicating the version of the input Python command is the expected Python version.```

I think you might have reversed the 'false/true' comments.